rc/filetype/nim.kak

```diff
@@ -37,8 +37,18 @@ add-highlighter shared/nim regions
 add-highlighter shared/nim/code default-region group
 add-highlighter shared/nim/triple_string region '([A-Za-z](_?\w)*)?"""' '"""(?!")' fill string
 add-highlighter shared/nim/raw_string region [A-Za-z](_?[A-Za-z])*" (?<!")"(?!") fill string
-add-highlighter shared/nim/string region (?<!'\\)"(?!') (?<!\\)(\\\\)*" fill string
-add-highlighter shared/nim/comment region '#?#\[' '\]##?' fill comment
+add-highlighter shared/nim/string region (?<!'\\)"(?!') (?<!\\)(\\\\)*" group
+add-highlighter shared/nim/comment region '#?#\[' '\]##?' group
+add-highlighter shared/nim/comment_line region '#?#[^\[]' $ group
+
+add-highlighter shared/nim/string/fill fill string
+add-highlighter shared/nim/comment/fill fill comment
+add-highlighter shared/nim/comment_line/fill fill comment
+# Escape sequences in string
+add-highlighter shared/nim/string/escape regex \\[prcnlftv] 0:value
+# Comment tags
+add-highlighter shared/nim/comment/tags regex \b(TODO|FIXME|XXX|NOTE|BUG)\b 0:default+rb
```

Comment:
This shouldn't be done by language plugin.

Reply:
Are you referring to both the escape sequences and the comment tags or just the tags? I know that there is [this](https://github.com/lenormf/kakoune-extra/blob/master/comnotes.kak) script for the tags, but it has a global scope and I couldn't get it to activate only on nim buffers (for some reason changing global/ to buffer/ completely disables ALL highlighters, even the ones in nim.kak).

Of course, if it's a kakoune policy I'll just remove it.

Reply:
these should be opt-in, since they are not related to language in any way. its a one liner, you can just add it to your own `hook global WinSetOption filetype=nim`.

Reply:
Ok. I removed them.